### DISCUSSION TITLE
Unable to use GitHub Copilot in VSCode - Invalid token and server connection issues.

### DISCUSSION THREAD

**Discussion by Caldeira888** [community/community#49046] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I’d like to report an issue I'm experiencing with the GitHub Copilot extension in VSCode. I've installed the extension and followed the setup instructions, and initially, it worked perfectly fine. However, the next day, without any apparent explanation, the extension stopped working, and I began encountering the following errors:

"Invalid copilot token: missing token: 403"
"GitHub Copilot could not connect to server. Extension activation failed: "No access to GitHub Copilot found.""

I've tried uninstalling and reinstalling the extension, as well as restarting VSCode, but the issue persists. I've also confirmed that my GitHub account is authorized to use Copilot.

**Comment by Caldeira888** [comment]
Guys, any other suggestions?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Caldeira888 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by pionear-layen** [comment]
Hey, guys, any luck with that？
